plugins/thunderbolt/fu-plugin-thunderbolt.c

```diff
@@ -62,6 +63,7 @@ fu_plugin_init (FuPlugin *plugin)
 	fu_plugin_add_udev_subsystem (plugin, "thunderbolt");
 	fu_plugin_set_device_gtype (plugin, FU_TYPE_THUNDERBOLT_DEVICE);
 	fu_plugin_add_firmware_gtype (plugin, "thunderbolt", FU_TYPE_THUNDERBOLT_FIRMWARE);
+	fu_plugin_add_firmware_gtype (plugin, "thunderbolt-update", FU_TYPE_THUNDERBOLT_FIRMWARE_UPDATE);
```

Comment:
Nice one. This is how I would have done it. The only thing that irks me is the name -- `update` isn't the best noun -- maybe `spi`?

Reply:
Well "thunderbolt" is the one from the SPI, "thunderbolt-update" is the one you're getting from the web.  So maybe instead:

"thunderbolt-spi" and "thunderbolt-update"?

Reply:
Or alternatively "thunderbolt-controller" and "thunderbolt-update"

Reply:
maybe `update` is fine then.